### PR TITLE
LPS-133262 Fix Questions app navigation

### PR DESCRIPTION
**Pull request by javierdearcos**
The href calculated by the buildURL method in Questions.es.js page has a starting "/" that the page don't have (i.e. http://localhost:8080/web/guest/questions-page#/questions/portal). I think this comes from the friendly URL that does not admit URLs ending with slash. Removing the initial slash in the buildUrl method fix the page navigation

### CODE REVIEW

**Comment by javierdearcos** [comment]
This fix solves also the [LPS-133231](https://issues.liferay.com/browse/LPS-133231) bug

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js [hunk 1]
```diff
@@ -370,7 +370,7 @@ export default withRouter(
 		]);
 
 		function buildURL(search, page, pageSize) {
-			let url = (context.historyRouterBasePath || '/#') + '/questions';
+			let url = (context.historyRouterBasePath || '#') + '/questions';
```
Comment (julien) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js:373]: Looks OK.
One thing we could do to validate the URL (if that's what we want of course) is to use the [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL) API

**Review by nhpatt — CHANGES_REQUESTED** [review]
Remember that there are more components with pagination...

**Comment by nhpatt** [comment]
ci:forward

**Comment by javierdearcos** [comment]
Sorry for missing the other cases...

**Pull request closed without merging**